packages/flutter_tools/lib/src/commands/analyze_base.dart

```diff
@@ -143,19 +149,22 @@ class PackageDependency {
   void describeConflict(StringBuffer result) {
     assert(hasConflict);
     final List<String> targets = values.keys.toList();
-    targets.sort((String a, String b) => values[b].length.compareTo(values[a].length));
+    targets.sort((String a, String b) => values[b]?.length ?? 0 .compareTo(values[a]?.length ?? 0));
```

Comment:
I think these should be `!.length` and not `?.length ?? 0`

Reply:
Thank you for your review. After analysis, `valuse [] `is not `null`, I have already modified the code in accordance with the recommendation.